packages/server/utils/analytics/analytics.ts

```diff
@@ -102,6 +102,10 @@ export type AnalyticsEvent =
   // snackbar
   | 'Snackbar Clicked'
   | 'Snackbar Viewed'
+  // Join request
+  | 'Join Request Reviewed'
+  | 'Join Request Denied'
+  | 'Join Request Accepted'
```

Comment:
+1 Would change this to match my previous feedback:
```suggestion
  | 'Join Request Viewed'
  | 'Join Request Reviewed'
```